dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -100,8 +179,11 @@ def SetUp(self,
             grr_username: str,
             grr_password: str,
             max_file_size: str,
-            approvers: Optional[str]=None,
-            verify: bool=True) -> None:
+            approvers: Optional[str] = None,
+            verify: bool = True,
+            match_mode: Optional[str] = None,
+            client_operating_systems: Optional[str] = None,
+            client_labels: Optional[str] = None) -> None:
```

Comment:
Actually, none of these arguments should be optional (we always provide the full list of kwargs, and ther default values are defined at recipe level). Some cleanup happened but apparently not entirely, heh. Could you remove the `Optional` checks and default values here?

Reply:
Cleaned up.